src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const litleDiscount = 20;
+  const bigDiscount = 50;
+  let amount = 0;
+  const priceMultDays = price * days;
+
+  if (days <= 2) {
+    return amount + priceMultDays;
+  }
+
+  if (days >= 3 && days <= 6) {
```

Comment:
```suggestion
  if (days <= 6) {
```
you've already checked that days >= 3 in previous if